BREAKING-CHANGES.md

```diff
@@ -182,8 +229,6 @@ The following example shows how to migrate for `Line`, but the same applies for
 
 ### Remove deprecated `Block::title_on_bottom` ([#757])
 
-[#757]: https://github.com/ratatui-org/ratatui/pull/757
```

Comment:
Why remove the link? Even when its duplicated somewhere its directly there to be easily clicked so I would keep it here.

Otherwise: put the full URL into the title?

Reply:
I'm not sure - likely some sort of auto formatting and I didn't notice. Will fix.

I avoid links in the title for shorter lines which are easier to avoid wrapping with in this context. Generally in prose refs rather than inline links lead to more readable markdown. Markdownlint detects missing links. Vscode removes unlinked refs, and highlights links as an error if they are missing corresponding refs. There's likely similar tooling for other editors.

Reply:
The removal of this was due to this being a duplicate incorrect link (the other one should have been 759, not 757). Markdown lint removed this as having two link definitions that point at links is potentially ambiguous.

Fixed in 1051d9db1